understanding/20/focus-order.html

```diff
@@ -134,7 +134,8 @@ <h2>Examples of Focus Order</h2>
          
          <li>A web page implements modal dialogs via scripting. When the trigger button is activated,
             a dialog opens and focus is set within the dialog. As
-            long as the dialog is open, focus is limited to the elements of the dialog. When the
+            long as the dialog is open, focus is generally limited to the elements of the dialog (though,
+            depending on the implementation, focus may also cycle through the user agent controls). When the
```

Comment:
```suggestion
            long as the dialog is open, all web page content outside the dialog becomes 
            <a href="https://html.spec.whatwg.org/multipage/interaction.html#inert">inert</a>
            and cannot receive focus. When the
```

How about sidestepping user agent controls and instead scoping to “_all web page content outside the dialog_” and then emphasizing the inertness of that content rather than trapping the user inside the dialog?

Reply:
i'd avoid linking directly to the html spec, as we generally try to be technology-agnostic (unless it's in a technology-specific technique, or it's given as a specific example for a particular technology). long story short, i'd keep "inert" but without the link around it.